spec/helpers/browser.ts

```diff
@@ -74,7 +74,9 @@ export async function close() {
  *          Default `5`
  */
 export async function generatePage(args: {
+  injection?: { beforeSnippet?: string };
   variantByFeature?: { [feature: string]: string };
+  variables?: { [key: string]: string };
   frameworks?: (keyof typeof FRAMEWORK_TAG_MAP)[];
```

Comment:
Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead to comply with https://google.github.io/styleguide/tsguide.html#arrayt-type